loadtest/utils.go

```diff
@@ -87,3 +89,19 @@ func nextPowerOf2(val int) int {
 	val++
 	return val
 }
+
+// RandomFutureTime generates a random time between (now + xMilliseconds) to (now + xDays) in Unix milliseconds.
```

Comment:
Nit: can we use the name of the variables themselves? It took me a while to understand the comment
```suggestion
// RandomFutureTime generates a random time between (now + millisecondsFromNow) to (now + uptoDaysFromNow) in Unix milliseconds.
```

Reply:
Done.